sagemaker-python-sdk/tensorflow_script_mode_mnist/tensorflow_distributed_mnist.ipynb

```diff
@@ -0,0 +1,224 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MNIST distributed training  \n",
```

Comment:
I believe we are moving towards adding examples explaining SageMaker features instead of specific datasets. I would remove this notebook specific features that you want to demonstrate.